src/components/list/ListForm.vue

```diff
@@ -167,27 +174,53 @@ export default {
     },
     addFilter(type) {
       this.list[type].push({
-        name: null,
+        title: null,
         id: null,
       });
       this.$nextTick(() => {
-        const refName = `${type}Input`;
+        const ref = `${type}Input`;
 
-        this.$refs[refName][this.$refs[refName].length - 1].focus();
+        this.$refs[ref][this.$refs[ref].length - 1].focus();
       });
     },
-    verifyFilters() {
-      const filtersNames = [];
+    isListTitleUnique() {
+      const isListTitleUnique = !this.lists.some(storeList => storeList.title === this.list.title);
+
+      this.errorMessage = isListTitleUnique ? '' : 'you already have a list with this title';
+
+      return isListTitleUnique;
+    },
+    areFiltersTitlesUnique() {
```

Comment:
it's also sounds like a computed, but if you need it as a method - it has to be a verb like "check" or something

Reply:
or validate